hpx/async_launch_policy_dispatch.hpp

```diff
@@ -66,7 +70,12 @@ namespace hpx { namespace detail
     ///////////////////////////////////////////////////////////////////////////
     template <typename Action>
     struct async_launch_policy_dispatch<Action,
-        typename std::enable_if<!traits::is_action<Action>::value>::type>
+#if defined(HPX_HAVE_DISTRIBUTED_RUNTIME)
+        typename std::enable_if<!traits::is_action<Action>::value>::type
+#else
+        void
+#endif
+        >
```

Comment:
Why is this specialization (for Actions) enabled if `HPX_HAVE_DISTRIBUTED_RUNTIME` is off?

Reply:
Same thing applies as in https://github.com/STEllAR-GROUP/hpx/pull/4091#discussion_r403913899. Thanks!